src/Gemini.Modules.Inspector/Inspectors/NoUndoEditorBase.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Gemini.Modules.Inspector.Inspectors
+{
+    /// <summary>
+    /// This class is used as an editor base for values that do not support
+    /// undo / redo operations, for example write only variables.
+    /// </summary>
+    /// <typeparam name="TValue">Type of the value</typeparam>
+    public abstract class NoUndoEditorBase<TValue> : EditorBase<TValue>
+    {
+        public override TValue Value
+        {
+            get { return (TValue) BoundPropertyDescriptor.Value; }
```

Comment:
This is the reason I asked about write-only - because this line appears to be reading the value?